project3/terraform/modules/rds-tfm/rds.tf

```diff
@@ -5,8 +5,8 @@ module "rds" {
   identifier        = "${var.application}-${var.environment}-${var.identifier}-rds"
   apply_immediately = var.db_apply_immediately
 
-  engine         = var.db_engine
-  engine_version = var.db_engine_version
+  engine               = var.db_engine
```

Comment:
Almost wondering if this was done intentionally as a test/tell to see if the code has actually been executed.

Reply:
I wonder :)